src/node_crypto.cc

```diff
@@ -3952,6 +3953,41 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment *env, v8::Local<v8::Object> options,
+                            int *padding, int *saltlen) {
+  Local<Value> key = String::NewFromUtf8(env->isolate(), "padding");
+  MaybeLocal<Value> maybePadding = options->Get(key);
+  if (!maybePadding.IsEmpty()) {
```

Comment:
Considering that I `return 0` after throwing an exception (so `0 <=> error`), I should probably `return 1` here as it should be optional to specify the padding. Apart from that, I agree.